src/features/pages/404-error/content/not-found.module.scss

```diff
@@ -0,0 +1,20 @@
+@use 'features/styles/theme/theme-mixins' as *;
+
+.not_found {
+    padding-top: 12rem;
+    @include breakpoints(tablet) {
+        padding-top: 8.6rem;
+        block-size: 90svh;
```

Comment:
![image](https://github.com/binary-com/deriv-com/assets/126637868/7abf90cd-ef2f-42b1-8ba6-0cc20abdf340)

```suggestion
        min-block-size: 90svh;
```

This will fix the problem

Reply:
sure Thank you